assets/css/v2/elevator/elevator_closures.scss

```diff
@@ -34,118 +61,238 @@
     opacity: 0.5;
   }
 
-  .outside-closure-list {
+  .subheading {
+    font-size: 80px;
+    font-weight: 700;
+    line-height: 80px;
+  }
+
+  .current-elevator-closed-view {
     position: relative;
     height: 100%;
-    background-color: $warm-neutral-90;
 
-    .header-container {
-      margin: 48px;
-      margin-bottom: 0;
+    .shape {
+      position: absolute;
+      top: 0;
+      right: 0;
+      display: inline-block;
+      border-right: 345px solid $accessibility-blue;
+      border-bottom: 300px solid transparent;
+    }
 
-      .header {
-        display: flex;
-        max-height: 432px;
-        margin-bottom: 48px;
-        font-size: 112px;
-        font-weight: 700;
-        line-height: 112px;
-
-        &__title {
-          word-spacing: 9999px;
+    .header {
+      box-sizing: border-box;
+      height: 528px;
+      padding: 53px 68px 51px 48px;
+      font-weight: 700;
+
+      .icons {
+        margin-bottom: 21px;
+
+        .no-service-icon {
+          margin-right: 16px;
         }
       }
+
+      .closed-text {
+        margin-bottom: 18px;
+        font-size: 150px;
+        line-height: 150px;
+      }
+    }
+
+    hr.thin {
+      margin: 0 20px;
     }
 
-    .closure-list-container {
-      overflow: hidden;
+    .accessible-path-container {
+      box-sizing: border-box;
+      height: 984px;
+      padding: 48px;
 
-      .closure-list {
+      .subheading-container {
         display: flex;
-        flex-flow: column wrap;
-        height: 904px;
-        transform: translateX(calc(-100% * var(--closure-list-offset)));
-
-        .closure-row {
-          box-sizing: border-box;
-          width: 1080px;
-          padding: 0 48px;
+        align-items: center;
+        justify-content: space-between;
+        height: 142px;
 
-          &__station-name {
-            font-size: 62px;
-            font-weight: 600;
-            line-height: 80px;
-            color: $cool-black-15;
-          }
+        .subheading {
+          align-self: center;
+        }
 
-          &__name-and-pills {
-            display: flex;
-            align-items: center;
-            margin-bottom: 14px;
+        .arrow {
+          width: 100px;
+          height: 100px;
+        }
+      }
 
-            .route-pill {
-              width: 132px;
-              height: 68.13px;
-              margin-right: 24px;
+      .alternate-direction-text {
+        color: $cool-black-30;
+      }
 
-              &__text {
-                line-height: 68.13px;
-              }
+      .alternate-direction-text.small {
```

Comment:
These could be nested to avoid the repetition, like

```scss
.alternate-direction-text {
  color: $cool-black-30;

  &.small {
    // ...
  }
```

Reply:
This was intentional due to some issues using the nesting selector when implementing the other layout. This is one of the places where having a working browserslist + stylelint configuration would be very helpful. I think I should add a comment to the top of this file communicating the modern styles that I know don't work on these screens.

Reply:
I'm not sure what browser-specific issues could exist here? Sass is going to compile things like nesting and `&` down to the equivalent "flat" CSS, unless something has changed and they've started using the (_very_ new) native CSS nesting spec.